maven/bnd-export-maven-plugin/src/test/resources/integration-test/test/export-from-inputbundles/pom.xml

```diff
@@ -0,0 +1,32 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>biz.aQute.bnd-test</groupId>
+        <artifactId>test</artifactId>
+        <version>0.0.1</version>
+    </parent>
+
+    <groupId>biz.aQute.bnd-test</groupId>
+    <artifactId>export-from-inputbundles</artifactId>
+    <version>0.0.1</version>
+    <packaging>pom</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-export-maven-plugin</artifactId>
+                <configuration>
+                    <bundlesOnly>true</bundlesOnly>
+                    <bundles>
+                        <bundle>bundles/org.apache.felix.eventadmin-1.4.8.jar</bundle>
```

Comment:
The relativity to project basedir is already implied for file collections. Would documenting that be enough?